install.sh

```diff
@@ -664,8 +675,7 @@ TasksMax=infinity
 TimeoutStartSec=0
 Restart=always
 RestartSec=5s
-ExecStart=$BIN_DIR/node_exporter \\
-    $CMD_NODE_EXPORTER_EXEC
+ExecStart=$BIN_DIR/node_exporter $CMD_NODE_EXPORTER_EXEC
```

Comment:
Modify Line `628`
**From**
```sh
command_args="$(escape_dq "$CMD_NODE_EXPORTER_EXEC")
    >>$LOG_FILE 2>&1"
```
**To**
```sh
command_args="$(escape_dq "$CMD_NODE_EXPORTER_EXEC") >> $LOG_FILE 2>&1"
```
The same behaviour of systemd must be reflect on openrc

Reply:
This isn't even part of my fixes in this PR. Please fix this on your master branch instead ;)